plugin/pluginConfig.go

```diff
@@ -37,7 +37,7 @@ type ConfigInterface interface {
 }
 
 type DumperPlugin struct {
-	configs     []v1.Plugin
+	plugin      []v1.Plugin
```

Comment:
If it is a slice, we would define this attribute as `plugins` rather than `plugin`. WDYT?

Reply:
ok thanks